idaes/power_generation/unit_models/helm/condenser_ntu.py

```diff
@@ -354,8 +366,8 @@ def initialize(
         )
 
         # Release Inlet state
-        blk.side_1.release_state(flags1, outlvl + 1)
-        blk.side_2.release_state(flags2, outlvl + 1)
+        blk.side_1.release_state(flags1, outlvl)
+        blk.side_2.release_state(flags2, outlvl)
         from_json(blk, sd=istate, wts=sp)
```

Comment:
Just a general comment that `hot_side`/`cold_side` and `side_1`/`side_2` are used interchangeably. Not sure if this is intended.